pkg/networking/networking.go

```diff
@@ -213,6 +221,19 @@ func (n *networkImpl) addDefaultHeader(request *http.Request) {
 			request.Header.Add(k, v[i])
 		}
 	}
+
+	ua := n.config.Get(configuration.USER_AGENT)
```

Comment:
Question: In general I'm still struggling to understand from a semantic perspective why user agent is a configuration value. Why didn't we just introduce something like `setUserAgent()`  to the NetworkAccess? This would provide a clear interface.

Reply:
Suggestion: To me this change seems unrelated to what the PR actually wants to achieve, so maybe separate things, so that you can get the task at hand resolved and merged and we can have architectural discussions and decisions outside and non-blocking.